modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js

```diff
@@ -252,6 +259,25 @@ const InviteTeamMembersPage = ({
 		}
 	};
 
+	const handleAddTeamMember = () => {
+		const emptyEmailAddress = values?.invites?.some(({email}) => !email);
+
+		setshowEmptyEmailError(emptyEmailAddress);
```

Comment:
Should be good choose a better variable name :)
A good practice it's always be a trusty value and negative it. Eg.: hasEmailsFilled, and add our negative value: !hasEmailsFilled